about.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styleAbout.css" type="css/text">
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=papyrus">
+    <title>We Care</title>
+    <style type="text/css">
+        .myheader{
+            padding: 10px;
+            background-color: black;
+        }
+    </style>
+   
+</head>
+<body style="background-image:url('img/banner_new.jpg');
```

Comment:
Inline styling is a bad idea. Make sure you look up these online and get back if you have any queries.